lib/stealth/services/messagebird/reply_handler.rb

```diff
@@ -117,7 +119,17 @@ def generate_buttons(buttons:)
             when 'url'
               "#{button['text']}: #{button['url']}"
             when 'payload'
-              "Para #{button['text'].downcase}: Texto #{button['payload'].upcase}"
+              message = I18n.t(
+                "stealth.messagebird.for",
+                default: "for"
+              )
+
+              text = I18n.t(
+                "stealth.messagebird.text",
+                default: "text"
+              )
+
+              "#{message.capitalize} #{button['text'].downcase}: #{text.capitalize} #{button['payload'].upcase}"
```

Comment:
Une façon de traduire des mélanges de textes et de variables est d’utiliser des variables de traduction.

Par exemple ici tu pourrais avoir : 

```suggestion
              I18n.t(
                "stealth.messagebird.text_option",
                text: button['text'],
                option: button['payload'],
                default: "For %{text}: Text %{option}",
              )
```

Ça permets d’avoir une seule clef, qui donnerait par exemple `Para %{text}: Texto %{option}`. Ça permets également de changer l’ordre des arguments si ça plus de sens dans une autre langue ou de changer la ponctuation, par exemple en ajoutant une espace avant le `:` comme on devrait le faire en français.

Reply:
@sunny génial!!

dans ce cas-ci, tu le laisserais comme cela?

```
suggestions.each_with_index do |suggestion, i|
              message = I18n.t(
                "stealth.messagebird.for",
                default: "for"
              )
              translated_reply = [
                translated_reply,
                "#{i + 1} #{message} #{suggestion}"
              ].join("\n")
            end
```

Reply:
Ah bien vu, dans ce cas tu pourrais également faire une clef qui contient des variables pour le numéro et la suggestion!

Ça permets par exemple de préfixer ou suffixer facilement dans l’application, par exemple en mettant comme traduction `Type %{number} for: ”%{suggestion}”`.